src/modules/icon/icon.css

```diff
@@ -0,0 +1,124 @@
+}
+
+.icon,
+.icon-info-outline,
+.icon-launch,
+.icon-label,
+.icon-chevron-left,
+.icon-expand-more,
+.icon-view,
+.icon-component,
+.icon-align-top,
+.icon-align-bottom,
+.icon-clear,
+.icon-align-right,
+.icon-align-left,
+:not(.ie8) .viewer-header-select-wrapper:before {
+    font-family: 'icon-font';
+    font-weight: normal;
+    font-style: normal;
+    line-height: normal;
+    font-size: 1.4rem;
+    vertical-align: sub; /*  ¯\_(ツ)_/¯ */
+}
+
+.icon-info-outline:before {
+    content: '\e639';
+}
+
+.ie8 .icon-info-outline {
+    width: 50px;
+}
+
+.ie8 .icon-info-outline:before {
+    font-family: sans-serif;
+    content: 'Info';
+    font-weight: 700;
+}
+
+.icon-launch:before {
+    content: '\e648';
+}
+
+.icon-label:before {
+    content: '\e887';
+}
+
+.ie8 .icon-label {
+    width: 95px;
+}
+
+.ie8 .icon-label:before {
+    font-family: sans-serif;
+    content: 'Annotations';
+    font-weight: 700;
+}
+
+.icon-chevron-left:before {
+    content: '\e8a7';
+}
+
+.icon-expand-more:before {
+    content: '\e8ab';
+}
+
+.icon-view:before {
+    content: '\e699';
+}
+
+.icon-component:before {
+    content: '\e628';
+}
+
+.icon-align-top:before {
+    content: '\e7a7';
+}
+
+.ie8 .icon-align-top {
```

Comment:
All IE8 styles can be removed. The Riot version used (2.3+) no longer supports IE8. When testing in IE8 the developer will just have to open the individual demo pages without the viewer.